samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/evaluation/TotalScoresBean.java

```diff
@@ -220,6 +232,53 @@ protected void init() {
 		
 		agents = newAgents;	
 	}
+
+  public boolean isOneSelectionType() {
+    if (this.getPublishedAssessment() != null) {
```

Comment:
if (this.getPublishedAssessment() == null) {
return false;
}

And continue with the implementation.
.....

Reply:
Ok